chrome/skin/style-osx.css

```diff
@@ -0,0 +1,7 @@
+@import 'style.css';
+
+#universal-search-recommendation {
+  background-color: Highlight;
+  color: HighlightText;
+  background-image: linear-gradient(rgba(255,255,255,0.3), transparent);
```

Comment:
Tiny little nitpick: would you mind swapping this line with `color` so the declarations are in alphabetical order?